### PR TITLE
Add some examples for testing

### PR DESCRIPTION
**Pull request by edwardloveall**
It took me a bit to figure out testing. It's quite simple, but an example would have saved me a lot of time. Changes/suggestions welcome 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +158,43 @@ end
...
+
+```ruby
+class UserPresenterTest < Keynote::TestCase
+  setup do
+    user = User.new(first_name: "Alice", last_name: "Smith")
+    @presenter = present(user)
+  end
+
+  test "display name" do
+    assert_equal @presenter.diaplay_name, "Alice Smith"
```
Comment (rf-) [inline README.md]: Typo: `display_name`

File: README.md [hunk 2]
```diff
@@ -158,6 +158,43 @@ end
 You can also generate prefixed methods like `user_first_name` by passing
 `prefix: true` to the `delegate` method.
 
+## Testing
+
+Testing Keynote is the same as using it in views or controllers. It can be tested with RSpec, Test::Unit, MiniTest, or MiniTestUnit.
```
Comment (rf-) [inline README.md]: I think the official spelling of the latter might be MiniTest::Unit

**Comment by rf-** [comment]
Awesome, thanks! That was kind of a weird oversight on my part. I'll just merge now and then do a followup commit to fix that spelling stuff. 👍

**Comment by edwardloveall** [comment]
Glad you merged it. Good catch on the typos. Cheers!

**Pull request merged**